test/ably/resthttp_test.py

```diff
@@ -135,3 +139,16 @@ def test_custom_http_timeouts(self):
         self.assertEqual(ably.http.http_open_timeout, 8)
         self.assertEqual(ably.http.http_max_retry_count, 6)
         self.assertEqual(ably.http.http_max_retry_duration, 20)
+
+    def test_request_headers(self):
```

Comment:
I realise it's not been done before, but with all new tests can you add a comment with the spec items it matches.  This makes it a lot easier to grep for tests